draft-ietf-quic-version-negotiation.md

```diff
@@ -387,6 +387,41 @@ lifetime of the connection; endpoints MUST NOT change the version that they
 consider to be in use based on the Version field of long headers as that field
 could be forged by attackers.
 
+# Minimum support
+
+Hosts can support this standard incrementally to simplify implementation, if
+not all features of the design are desired.
+
+{{INV}} and {{QUIC}} already describe how servers send Version Negotiation
+packets in response to long header packets with unsupported version fields, even
+if they support only one version. Doing so prevents long timeouts at the client,
+but servers might not send the packet multiple times for the same connection, or
+when under stress.
+
+If a server supports two or more QUIC versions and sends Version Negotiation
+packets, it MUST follow the requirements in {{vers-info}} and {downgrade}}.
+Failure to comply would leave the connection vulnerable to downgrade attacks, or
+cause connection failure, if the client initiates Incompatible Version
+Negotiation ({{incompat-vn}}).
+
+A server that supports two or more compatible QUIC versions MAY implement
+Compatible version negotiation ({{compat-vn}}) if it prefers one version over
+another.
```

Comment:
I would rewrite this almost entirely non-normatively by describing what is technically possible.

E.g.,

It is possible to implement only VN or compatible version negotiation. Servers which implement only compatible version negotiation will not be able to interoperate with clients which send compatible Initial values. Servers which implement only VN will not be able to select between multiple acceptable versions [I think this is true.]

Reply:
OK